core/src/database/migrations.cpp

```diff
@@ -976,7 +976,7 @@ namespace ledger {
                        "PRIMARY KEY (idx, transaction_uid)"
                        ")";
                 sql << "INSERT INTO bitcoin_outputs_swap "
-                       "SELECT idx, transaction_uid, transaction_hash, amount, scripts, "
+                       "SELECT idx, transaction_uid, transaction_hash, amount, script, "
```

Comment:
Was that migration released yet? If so, we shouldn’t update migrations like that, as it will go full Kitchen Nightmare™ on production (since they have already applied the migration).